editor/editor_themes.cpp

```diff
@@ -825,6 +827,9 @@ Ref<Theme> create_editor_theme(const Ref<Theme> p_theme) {
 	theme->set_color("font_hover_color", "MenuButton", font_hover_color);
 	theme->set_color("font_hover_pressed_color", "MenuButton", font_hover_pressed_color);
 	theme->set_color("font_focus_color", "MenuButton", font_focus_color);
+	theme->set_color("font_outline_color", "MenuButton", font_outline_color);
+
+	theme->set_constant("outline_size", "MenuButton", 0 * EDSCALE);
```

Comment:
```suggestion
	theme->set_constant("outline_size", "MenuButton", 0);
```

Reply:
I used the `0 * EDSCALE` here to be consistent with the rest, e.g. `check_v_offset`, which has also this style.
But I'm also fine with changing it, then I would alos change `check_v_offset` to be consistent.